util.c

```diff
@@ -370,3 +367,28 @@ void free_masquerade_settings(struct masquerade_config_t *masquerade)
 		free(masquerade->host);
 	free(masquerade);
 }
+
+/* Copy of errlog[x] to print a warning in a consistent format
+ * as this is just a warning, it doesn't exit
+ */
+void log_warning(const char *fmt, ...)
+{
+        va_list ap;
+        char outs[ERRMSG_SIZE];
+
+        outs[0] = 0;
+        if (fmt != NULL) {
+                va_start(ap, fmt);
+                vsnprintf(outs, sizeof(outs), fmt, ap);
+                va_end(ap);
+        }
+
+        if (*outs != 0) {
+                syslog(LOG_WARNING, "%s", outs);
+                fprintf(stderr, "%s: %s\n", getprogname(), outs);
+        } else {
+                syslog(LOG_WARNING, "Unknown warning");
```

Comment:
This is a useless message.  When would this ever happen?  I guess if you want to log an empty line?  In that case, we shouldn't convert it to "Unknown warning".

Reply:
You're right, this is probably as useless here as in the existing errlog functions. Nobody should call these functions with NULL as format, or even funnier with something like "char bla ='\0'; errlog(666, &bla);"
So somebody should probably remove these checks in those functions as well.

Reply:
I agree.